assets/src/features/leaving-page/icons/🚀.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+const Rocket: React.FC<React.SVGAttributes<SVGElement>> = (props) => {
+  return (
+    <svg
+      width="200"
+      height="200"
+      viewBox="0 0 200 200"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={props.className}
+      {...props}
+    >
+      <path
+        d="M126.064 43.0331C115.502 44.6312 78.0055 53.75 77.9899 102.197L102.327 116.155C144.135 91.6787 133.657 55.0945 129.867 45.2142C129.591 44.4656 129.059 43.8372 128.364 43.4386C127.669 43.04 126.855 42.8965 126.064 43.0331Z"
+        fill="#D2CFFD"
+      />
+      <path
+        d="M108.602 77.4435C111.122 78.889 114.345 78.0312 115.8 75.5275C117.255 73.0239 116.391 69.8225 113.871 68.377C111.351 66.9316 108.128 67.7894 106.673 70.293C105.218 72.7967 106.081 75.998 108.602 77.4435Z"
+        fill="#001A72"
+      />
+      <path
+        fill-rule="evenodd"
+        clip-rule="evenodd"
```

Comment:
Fix properties in svg to jsx compatible
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```